documentation-site/static/examples/table-grid/nested.js

```diff
@@ -0,0 +1,265 @@
+        'running',
+        new Date(2019, 6, 22),
+        'Passed (6 minutes, 44 seconds)',
+        'https://buildkite.com/uberopensource/baseui/builds/7728#54927bc9-88e0-4d0b-80b0-f60b8a2298e4',
+      ],
+      [
+        'buildkite/baseui/docker-package-unit',
+        'passed',
+        new Date(2019, 6, 22),
+        'Passed (7 minutes)',
+        'https://buildkite.com/uberopensource/baseui/builds/7728#fea8c317-b65b-4c5f-9fad-b7a329a26237',
+      ],
+      [
+        'buildkite/baseui/documentation-site-link-checker',
+        'failed',
+        new Date(2019, 6, 22),
+        'Passed (1 minute, 33 seconds)',
+        'https://buildkite.com/uberopensource/baseui/builds/7728#eaa5998f-69a4-4b94-9a9b-4dd2bbd8c985',
+      ],
+      [
+        'buildkite/baseui/eslint',
+        'passed',
+        new Date(2019, 6, 22),
+        'Passed (1 minute, 34 seconds)',
+        'https://buildkite.com/uberopensource/baseui/builds/7728#a658a503-472f-43aa-80b5-708d39951cf7',
+      ],
+      [
+        'buildkite/baseui/flowtype',
+        'passed',
+        new Date(2019, 6, 22),
+        'Passed (1 minute, 33 seconds)',
+        'https://buildkite.com/uberopensource/baseui/builds/7728',
+      ],
+      [
+        'buildkite/baseui/jest',
+        'running',
+        new Date(2019, 6, 22),
+        'Passed (1 minute, 33 seconds)',
+        'https://buildkite.com/uberopensource/baseui/builds/7728',
+      ],
+      [
+        'buildkite/baseui/pipeline',
+        'running',
+        new Date(2019, 6, 22),
+        'Passed (1 minute, 33 seconds)',
+        'https://buildkite.com/uberopensource/baseui/builds/7728',
+      ],
+      [
+        'buildkite/baseui/screener',
+        'running',
+        new Date(2019, 6, 22),
+        'Passed (1 minute, 33 seconds)',
+        'https://buildkite.com/uberopensource/baseui/builds/7728',
+      ],
+    ],
+  ];
+}
+
+const data = [
+  buildRow('running'),
+  buildRow('running'),
+  buildRow('failed'),
+  buildRow('passed'),
+  buildRow('passed'),
+  buildRow('passed'),
+  buildRow('passed'),
+  buildRow('failed'),
+  buildRow('failed'),
+  buildRow('failed'),
+  buildRow('passed'),
+  buildRow('passed'),
+  buildRow('passed'),
+  buildRow('passed'),
+  buildRow('passed'),
+];
+
+function statusToTagKind(status) {
+  switch (status) {
+    case 'running': {
+      return 'primary';
+    }
+    case 'passed': {
+      return 'positive';
+    }
+    case 'failed': {
+      return 'negative';
+    }
+    default: {
+      return 'neutral';
+    }
+  }
+}
+
+// sticky elements create a new stacking context. because parent table head cells are _siblings_
+// of these visually child elements, stacking is determined by order in html. this leads to
+// unintended layering upon scrolling the outer table. this table should not need to scroll
+// therefore chose to remove sticky rather than provide relative zindex values.
+const NotStickyHeadCell = withStyle(StyledHeadCell, {
```

Comment:
could this be a prop or configured somehow? seems like it might be common enough use case

Reply:
I hesitate to prematurely add features that have not been necessary in-practice. I'd like to ship as-is and improve where requested

Reply:
I think I was too quick to write this off. I'm going to make it sticky by default with the option to turn off

Reply:
updated 👍